insights/core/spec_factory.py

```diff
@@ -268,7 +268,10 @@ def create_args(self):
         if self._filters:
             log.debug("Pre-filtering %s", self.relative_path)
             args.append(
-                ["grep", "-F", "\n".join(sorted(self._filters.keys(), reverse=True)), self.path]
+                ["tail", "-n", "1000000", self.path]
```

Comment:
It would be good to declare a GLOBAL constant for the line value, it's useful for documentation and future update.